group_vars/gxconfig.yml

```diff
@@ -320,11 +320,39 @@ base_app_main: &BASE_APP_MAIN
   # option.
   #watch_tours: 'false'
 
+  # Location of files available for a short time as downloads (short
+  # term storage). This directory is exclusively used for serving
+  # dynamically generated downloadable content. Galaxy may uses the
+  # new_file_path parameter as a general temporary directory and that
+  # directory should be monitored by a tool such as tmpwatch in
+  # production environments. short_term_storage_dir on the other hand is
+  # monitored by Galaxy's task framework and should not require such
+  # external tooling.
+  # The value of this option will be resolved with respect to
+  # <cache_dir>.
+  #short_term_storage_dir: short_term_web_storage
```

Comment:
Interesting. I would assume this directory can grow big and should be on the `jwd` not under __cache_dir__. Something to watch out.